update/ansible/playbook/tasks/update.yml

```diff
@@ -190,6 +190,10 @@
         - skip_ansible_lint # '503 Tasks that run when changed should likely be handlers'.
           # The handler looks bad in this case
 
+    - name: Cleanup yum metadata
+      command: yum clean metadata
+      become: true
```

Comment:
FYI: the superuser is already set at the very top of this playbook - https://github.com/percona/pmm/pull/2253/files#diff-9b5b8bbc4dded267d928c642902400dc7aec9b65e3812a8659cc4abf211e226dL4